src/Components/WebAssembly/WebAssembly/test/WebAssemblyHotReloadTest.cs

```diff
@@ -1,29 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The test doesn't add much value. The agent code is already tested in SDK repo. Once we share the code we would have full coverage coming from there.